src/models/spaces/organisms/organisms-row.ts

```diff
@@ -30,36 +30,52 @@ export const OrganismsRowModel = types
     hoveredOrganelle: types.maybe(Organelle),
     selectedOrganelle: types.maybe(Organelle),
     mode: types.optional(Mode, "normal"),
-    assayedOrganelle: types.maybe(Organelle),
+    assayedOrganelles: types.array(Organelle),
     zoomLevel: types.optional(ZoomLevel, "organism"),
     showProteinDNA: false,
     showProteinAminoAcidsOnProtein: false,
     proteinSliderStartPercent: 0
   })
   .views(self => ({
     get currentData(): ChartDataModelType {
-      const chartDataSets = [];
-      const points: DataPointType[] = [];
-      const organelle = self.assayedOrganelle;
-      if (organelle) {
-        kSubstanceNames.forEach((substance) => {
+      const chartDataSets: any[] = [];
+      const organelleLabels: string[] = [];
+
+      self.assayedOrganelles.forEach((organelle) => {
+        organelleLabels.push(organelle);
+      });
+      const colorMaps: any = { pheomelanin: "#f4ce83",
```

Comment:
Just learned you could do this inline in Sam's PR, I'd try a type def like:

```
const colorMaps: {[sub in SubstanceType]: string} = {...
```

Then, if we add any substances in the future, this map will automatically complain if it doesn't have a color associated with it.

Reply:
I yanked `colorMaps` entirely now that I'm getting colors out of the `kSubstanceType` that I created.